garden_ai/gardens.py

```diff
@@ -367,29 +455,6 @@ def __dir__(self):
         # this gets us jupyter/ipython/repl tab-completion of pipeline names
         return list(super().__dir__()) + self.pipeline_names
```

Comment:
since I'm thinking we should cut the `self.pipeline_names` attribute:
```suggestion
        pipeline_names = [self.pipeline_aliases.get(p.doi) or p.short_name for p in self.pipelines]
        return list(super().__dir__()) + pipeline_names
```